test/conftest.py

```diff
@@ -184,27 +186,53 @@ def ferc1_dbf_sql_engine(
     return engine
 
 
-@pytest.fixture(scope="session", name="ferc1_engine_xbrl")
-def ferc1_xbrl_sql_engine(
+@pytest.fixture(scope="session")
+def ferc_xbrl(
     pudl_settings_fixture,
     live_dbs,
     ferc_to_sqlite_settings,
     pudl_datastore_fixture,
 ):
-    """Grab a connection to the FERC Form 1 DB clone.
+    """Extract XBRL filings and produce raw DB+metadata files.
 
-    If we are using the test database, we initialize it from scratch first. If we're
-    using the live database, then we just yield a conneciton to it.
+    Extracts a subset of filings for each form for the year 2021.
     """
     if not live_dbs:
```

Comment:
This fixture just ensures that the XBRL extraction process happens, which means the DB and all metadata files should be in the right place. If `live_dbs` is true it's basically on the user to make sure that those files exist in the right places. Perhaps it would make sense to `assert` that all the files exist where expected so the tests fail early if they don't?

Reply:
I think that makes sense! Might even put that check in the `live_dbs` fixture